tools/java_api/src/main/java/com/kuzudb/KuzuValue.java

```diff
@@ -74,6 +74,7 @@ public void destroy() throws KuzuObjectRefDestroyedException {
             KuzuNative.kuzu_value_destroy(this);
             destroyed = true;
         }
+        System.gc();
```

Comment:
Leave a comment if we decide to keep this.

Reply:
Forgot about it. It's covered well by my other change so I'll delete this in the morning.